pkg/driver/kubernetes/run.go

```diff
@@ -164,8 +172,8 @@ func (k *kubernetesDriver) runContainer(
 	}
 
 	labels := map[string]string{}
-	if podTemplate != nil && podTemplate.ObjectMeta.Labels != nil {
-		for k, v := range podTemplate.ObjectMeta.Labels {
+	if pod.ObjectMeta.Labels != nil {
+		for k, v := range pod.ObjectMeta.Labels {
```

Comment:
Thats not really making sense, we apply the labels on the same object?

Reply:
how would you do it otherwise? 
we're merging all labels with the order `pod manifest` < `labels option` < `devpod labels`, if we'd append it to the existing pod template labels we would run into the risk of duplicated labels